test/functional/test_framework/mininode.py

```diff
@@ -372,7 +372,7 @@ def on_version(self, message):
     # Connection helper methods
 
     def wait_until(self, test_function, timeout):
-        wait_until(test_function, timeout=timeout, lock=mininode_lock, timeout_factor=self.timeout_factor)
+        wait_until(test_function, timeout=timeout, lock=mininode_lock)
```

Comment:
This belongs in the other commit?

Generally a typo isn't worth a separate commit. As you spend more time reviewing PRs, you'll see these practices.

Reply:
Right, it's all mixed up. Fixed now.

Reply:
This breaks the factor argument. You need to call `wait_until()` with `timeout = timeout * self.timeout_factor`.

Reply:
Agreed, fixed.